showup/management/commands/pull_seatgeek_data.py

```diff
@@ -107,14 +87,8 @@ def handle(self, *args, **options):
                         "I just saved event " + str(concert["id"]) + " to the database"
                     )
 
-        Genre.objects.all().delete()
-        # Sort genres alphabetically.
-        genres = sorted(existing_genres)
-
-        # Add genres to Genre model.
-
-        for genre in genres:
-            Genre(genre=genre).save()
-
-        # Add genres to genres.txt.
-        Command.WriteGenre(genres)
+                    # we have to create the Concert object before adding its genres
```

Comment:
where are you making a Concert object here?

Reply:
Lines 69-77 and 85

Reply:
ok thx